app/viewModels/taskList/TaskListViewModel.scala

```diff
@@ -22,7 +22,18 @@ case class TaskListViewModel(tasks: Seq[TaskListTask], submissionState: Submissi
 
   def showErrorContent: Boolean = submissionState.showErrorContent
 
-  def showSubmissionButton: Boolean = tasks.forall(_.isCompleted)
+  def showSubmissionButton: Boolean =
+    submissionState match {
+      case SubmissionState.Amendment          => containsAmended && allCompletedOrAmended
```

Comment:
could probably change this to 
```suggestion
      case SubmissionState.Amendment | SubmissionState.GuaranteeAmendment         => containsAmended && allCompletedOrAmended
```